### PR TITLE
Added Product Page With Filters

### PR DESCRIPTION
**Pull request by aniketxparihar**
Added Pages
Home
Login
Signup
Forgot Password

Added Components
Navbar
Footer
CategoryCard

### CODE REVIEW

**Review by JowelTisso — APPROVED** [review]
* Clean code 
* Looks good

File: src/pages/Filter/Filter.jsx [hunk 1]
```diff
@@ -10,4 +10,184 @@ const Filter = (dispatch) => {
   );
 }
 
+export default Filterimport React from 'react'
```
Comment (xsaurabhsuthar) [inline src/pages/Filter/Filter.jsx:13]: is this a typo?

**Review by xsaurabhsuthar — APPROVED** [review]

File: src/components/CheckboxInput/CheckboxInput.jsx [hunk 2]
```diff
@@ -3,9 +3,19 @@ import React from "react";
 const CheckboxInput = ({ name, className, id, label_text, value, checked }) => {
   return (
     <div className="category flex justify-center align-center">
-     
+      <input
+        type="checkbox"
+        id={id}
+        name={name}
+        className={className}
```
Comment (saketkum11) [inline src/components/CheckboxInput/CheckboxInput.jsx:10]: This good way to pass props to attributes

**Review by saketkum11 — APPROVED** [review]
- clean code 
- Looks good 
- UI is great.

**Pull request merged**